### DISCUSSION TITLE
Welcome to AirSim Discussions!

### DISCUSSION THREAD

**Discussion by guadacasuso** [microsoft/AirSim#3218] [General]
## 👋 Welcome to Airsim discussions!

  We’re using Discussions as a place to connect with other members of our community. 

We hope that you: 

  * Ask questions you’re wondering about. 

  * Share ideas. 

  * Engage with other community members. 

  * Welcome others and are open-minded. Remember that this is a community we  build together 💪. 

  To get started, comment below with an introduction of yourself and tell us about what are your expectations about this community.

**Comment by AngelTang190** [comment]
Hi, I am very new to AirSim and I hope via this platform I can get to learn more and can familiarize myself in using AirSim. I will really appreciate it if I can know whether there is a rotation matrix to convert local AirSim coordinates to global coordinates and vice versa? Thank you very much!

**Reply by jonyMarino** [reply]
Welcome, @AngelTang190! And thank you for your post! Please ask that question in the "Support Q&A" section. Thanks!
